components/03-sections/12-timeline/timeline.hbs

```diff
@@ -0,0 +1,109 @@
+<section class="timeline-container">
+  <div class="container py-5">
+    <h2>{{ headline }}</h2>
+    <div class="main-timeline">
+      <div class="col-12 timeline-banner-start"><h3>{{ subhead }}</h3></div>
+      <div class="timeline timeline-right">
+        <div class="card">
+          <div class="card-body p-4">
+            <h3 class="timeline-section-header">1. Section Header</h3>
+            <div class="row">
+              <div class="col-xs-12 col-md-6">
+                  <ul>
```

Comment:
On small devices, the two **ul** will stack into one column. They have space between them unless you remove the bottom margin from the first **ul**.

<img width="930" alt="Screen Shot 2024-10-30 at 6 36 28 PM" src="https://github.com/user-attachments/assets/82016991-7225-4f19-9f09-450a1ac535d1">

Reply:
agreed, @Uri-E, can you adjust the spacing on mobile view for these stacking items?